src/components/ButtonPanel.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import Button from './Button';
+
+const rowStyle = {
+  display: 'flex',
+  height: 100,
+};
+
+const ButtonPanel = () => (
+  <div id="button-panel">
+    <div id="group-1" style={rowStyle}>
+      <Button name="AC" color="light-gray" />
+      <Button name="+/-" color="light-gray" />
+      <Button name="%" color="light-gray" />
+      <Button name="÷" />
+    </div>
+    <div id="group-2" style={rowStyle}>
+      <Button name="7" color="light-gray" />
+      <Button name="8" color="light-gray" />
+      <Button name="9" color="light-gray" />
+      <Button name="X" />
+    </div>
+    <div id="group-3" style={rowStyle}>
+      <Button name="4" color="light-gray" />
+      <Button name="5" color="light-gray" />
+      <Button name="6" color="light-gray" />
+      <Button name="-" />
+    </div>
+    <div id="group-3" style={rowStyle}>
+      <Button name="4" color="light-gray" />
```

Comment:
[Optional]

We could rewrite the ButtonPanel component to use two loops to make the `Button` instead of hardcoding them. By doing this, we keep our code [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself).

We could achieve this by creating an array of symbol and loop through the array using [`map`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map) to create the ButtonPanel component.